etc/emqx.conf

```diff
@@ -458,7 +458,29 @@ log.file = emqx.log
 
 ## Log formatter
 ## Value: text | json
-#log.formatter = text
+log.formatter = text
+
+## Format of the text logger.
+##
+## Value: rfc3339 | FORMAT
+## Where FORMAT is the format string of the timestamp. Supported specifiers:
+##     %Y: year
+##     %m: month
+##     %d: day
+##     %H: hour
+##     %M: minute
+##     %S: second
+##     %N: nanoseconds (000000000 - 999999999)
+##     %6N: microseconds (00000 - 999999)
+##     %3N: milliseconds (000 - 999)
+##     %z: timezone, [+-]HHMM
+##     %:z: timezone, [+-]HH:MM
+##     %::z: timezone, [+-]HH:MM:SS
+##
+## For example:
+## log.formatter.text.date.format = %Y-%m-%dT%H:%M:%S.%6N %:z
```

Comment:
For example:
  log.formatter.text.date.format = %Y-%m-%dT%H:%M:%S.%6N %:z
  log.formatter.text.date.format = %Y-%m-%d %H:%M:%S.%3N